ada_feeding_perception/ada_feeding_perception/food_on_fork_logistic_reg_training.py

```diff
@@ -0,0 +1,50 @@
+import pandas as pd
+import numpy as np
+from sklearn.linear_model import LogisticRegression
+import joblib
+
+
+def main():
```

Comment:
Clearly this training function requires very different than the other one. That is why docstrings are crucial, for people to understand details like that.